### DISCUSSION TITLE
I had 2 answers accepted and didn't win the achievement

### DISCUSSION THREAD

**Discussion by onike14** [community/community#138116] [Other Feature Feedback, Questions, & Ideas] [answered]
### Body

I had 2 answers accepted in discussions in I didn't get the achievement that asks for this.

**Comment by MarkosComK** [comment]
It can take some time for GitHub to register achievements like the "Accepted Answer" badge in Discussions. If you've had two answers accepted and still haven't received the achievement, here are a few things to consider:

- Check if the discussions are in public repositories: Achievements are only awarded for activity in public repositories. If your answers were accepted in private repositories, they won’t count towards the badge.

- Wait for a delay in processing: Sometimes, GitHub’s system takes a while to update achievements. It can take up to 24 hours for the badge to appear.

- Contact GitHub Support: If after a few days the achievement still doesn't show up, you can try reaching out to GitHub Support for help.

**Comment by davevad93** [comment, marked as answer]
Hi @onike14 ,

Achievements are no longer available in the Community, for more info see here #106536.

You can still earn the `Galaxy Brain` badge by answering discussions outside the Community, e.g. in public repositories with open discussions.

**Reply by callmesidhu** [reply]
Thanks you
